extensions/PhabBugz/lib/Feed.pm

```diff
@@ -378,11 +378,25 @@ sub readable_answer {
 }
 
 sub format_uplift_request_as_markdown {
-  my ($question_answers_mapping) = @_;
+  my ($repo_short_name, $question_answers_mapping) = @_;
+
+  # Form content will come across a JSON object. Ensure the question/response pairs
+  # are added to the markdown output in the correct order.
+  my @uplift_questions_order = (
+    "User impact if declined",
+    "Code covered by automated testing",
+    "Fix verified in Nightly",
+    "Needs manual QE test",
+    "Steps to reproduce for manual QE testing",
+    "Risk associated with taking this patch",
+    "Explanation of risk level",
+    "String changes made/needed",
+    "Is Android affected?",
+  );
```

Comment:
Only issue I can see if additional fields are added in Phabricator, they will not automatically show up in the Bugzilla comment due to the hard coded list. We would just need to make sure to update this code as well if additional questions are ever added in Phabricator.

Reply:
Ideally we could have the output come across as an ordered list of `(question, answer)` pairs or something along those lines, but I don't want to muck about with the Phabricator code and running Phab in suite is broken at the moment. For now this is a quick-and-dirty solution. :)